server/tserver/src/main/java/org/apache/accumulo/tserver/TabletClientHandler.java

```diff
@@ -243,6 +245,8 @@ public void loadFiles(TInfo tinfo, TCredentials credentials, long tid, String di
             server.removeBulkImportState(files);
           }
         }
+        UtilWaitThread.sleep(100);
```

Comment:
Addes this sleep so that I could verify things were running concurrently in the tserver and never saw that happen.